fbw-a380x/src/systems/instruments/src/SD/SystemDisplay.tsx

```diff
@@ -30,40 +33,42 @@ import { useArinc429Var, useUpdate } from '@flybywiresim/fbw-sdk';
 enum SdPages {
   None = -1,
   Eng = 0,
-  Bleed = 1,
-  Press = 2,
-  ElecAc = 3,
-  Fuel = 4,
-  Hyd = 5,
-  Cb = 6,
-  Apu = 7,
-  Cond = 8,
-  Door = 9,
-  ElecDc = 10,
-  Wheel = 11,
-  Fctl = 12,
-  Video = 13,
-  Crz = 14,
-  Status = 15,
+  Apu = 1,
```

Comment:
Note that as discussed on Discord, changing this enum currently brakes the automatic SD page switching from `FwsAbnormalSensed`, as pages are referenced by ID there.

Reply:
Covered now - ready to review